SharedSequence/src/test/kotlin/org/notests/sharedsequence/utils/Events.kt

```diff
@@ -0,0 +1,17 @@
+package org.notests.sharedsequence.utils
+
+/**
+ * Created by juraj begovac on 13/9/18
+ */
+sealed class Event {
```

Comment:
Why do you use `Event` instead of `Notification`?

Reply:
will use notification and the you can see what looks better :)

Reply:
You don't have to :) I know that probably this looks better. It felt odd because it's a 1-1 mapping with `Notification`s :)

Reply:
Still I think it's ok to remove it :)